### ISSUE TITLE
Interested to know where you got the bad domain list

### ISSUE THREAD

**Issue by bbashy** [an-d-uu/pihole-sinkhole#1]
Hi,

Interested to know where you got the list for `probably_bad_domains` as my domain is listed https://github.com/an-d-uu/pihole-sinkhole/blob/master/probably_bad_domains#L6552

Thanks.

**Comment by an-d-uu** [comment]
This list came from various places, one being an old windows app so called spyware doctor that would enter the entries if found in your host file. These values were the experimental ones that I pulled before the app version I used was shutdown in favor their ad paid version.I have never been sure about these and do not actively use them because I have found discrepancies over the years. I am more than happy to remove any domains that are for sure good ones. It could have been at one point your domain got on that list before you owned it or it was a mistake. Just let me know and I'll remove it and make sure it's not on any of my other ones!
